Infrastructure.Data/Mapping/CustomerMapping.cs

```diff
@@ -0,0 +1,56 @@
+﻿using DomainModels.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Data.Mapping
+{
+    internal class CustomerMapping : IEntityTypeConfiguration<Customer>
+    {
+        public void Configure(EntityTypeBuilder<Customer> builder)
+        {
+            builder.ToTable("customer");
```

Comment:
Plural porque é uma tabela de clientes e não apenas 1
```suggestion
        {
            builder.ToTable("Customers");
```